packages/unpublished-test-infra/index.js

```diff
@@ -9,11 +9,9 @@ const addonBaseConfig = addonBuildConfigForDataPackage(name);
 
 module.exports = Object.assign({}, addonBaseConfig, {
   treeForAddon() {
-    if (process.env.EMBER_CLI_TEST_COMMAND) {
-      const options = this.getEmberDataConfig();
-      let compatVersion = options.compatWith;
-      let tree = version(compatVersion);
-      return this.debugTree(this._super.treeForAddon.call(this, tree), 'addon-output');
-    }
+    const options = this.getEmberDataConfig();
+    let compatVersion = options.compatWith;
+    let tree = version(compatVersion);
+    return this.debugTree(this._super.treeForAddon.call(this, tree), 'addon-output');
```

Comment:
So this leaks the version module into our asset-size check because we now include it in production builds from unpublished-test-infra.

I don't think it legitimately affects end consumers, but it is frustrating to have it included in the check when it should not be.

There's probably a check we could do here, but it wouldn't just be "is production" since we test production output too. Maybe we check if the command is "serve" ? @rwjblue thoughts?

Reply:
> we now include it in production builds from unpublished-test-infra.

That is a great point!  I assumed from the name of this package that it was not included in a published prod build; while also noticing it was a `devDependency` for various pkgs.  Side question - is there a short explanation or module that you could point me to in order to understand why `unpublished-test-infra` would be included in the process of a production build?

Also, would an alternative fix involve something like `process.env.EMBER_ENV`?